TddShop/TddShop.Cli/Shipment/RomanConverter.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TddShop.Cli.Shipment
+{
+    //Standard roman notation ends on 4000
+    //Notation with visculum extends system
+    //Here are sources for info on roman notations:
+    //http://www.numericana.com/answer/roman.htm
+    //https://en.wikipedia.org/wiki/Roman_numerals#Vinculum
+    //Main part
+    public partial class RomanConverter : IRomanConverter
```

Comment:
Fajnie napisana klasa, bardzo rozszerzalna ale utrudniało to kod w analizie, tak jak napisał @mturzynskigoyello warto by rozbić go na parę klas i rozwiązać bardziej obiektowo.